docs/Cookbook/Deploy/idxrc.php

```diff
@@ -19,9 +24,9 @@
 
 return \Idephix\Config::fromArray(
     array(
-        \Idephix\Config::ENVS => $environments,
-        \Idephix\Config::SSHCLIENT => new \Idephix\SSH\SshClient(new \Idephix\SSH\CLISshProxy()),
-        \Idephix\Config::EXTENSIONS => array(
+        'envs' => $environments,
+        'ssh_client' => new \Idephix\SSH\SshClient(new \Idephix\SSH\CLISshProxy()),
```

Comment:
is `CLISshProxy` an optional dependency? Asking since in the doc seems it's not needed